app/controllers/auth_controller.rb

```diff
@@ -1,4 +1,10 @@
-# Controller for the DAHLIA homepage.
-class AuthController < HomeController
+# Controller for the authentication pages.
+class AuthController < ApplicationController
   def sign_in; end
+
+  protected
```

Comment:
do you need to add `protected` to the HomeController, too?

Reply:
yes, method types should be consistent, good catch. Will add it.